extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -843,7 +802,7 @@ public function tabs() {
 		$config = $this->getVal('config', false);
 
 		if ( ! $config || (! $config instanceOf Wikia\Search\Config ) ) {
-		    throw new Exception("This should not be called outside of self-request context.");
+			throw new Exception("This should not be called outside of self-request context.");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS112)